cli/src/pcluster/constants.py

```diff
@@ -159,3 +159,5 @@
 NODEJS_INCOMPATIBLE_VERSION_RANGE = ["13.0.0", "13.6.0"]
 
 NODE_BOOTSTRAP_TIMEOUT = 1800
+
+PLUGIN_INTERFACE_VERSION = "1.0"
```

Comment:
minor - are we ok with PLUGIN_ prefix or should it be SCHEDULER_PLUGIN?

Reply:
Changed